jproject-fr/easy-eat
# #K589: Fix bug all buttons are loading separately

TO TEST:
- go to /poc/category
- Check if only one Add to cart button is loading. 
![image](https://github.com/jproject-fr/easy-eat/assets/117679829/11b314be-b4be-42e1-b05c-8e435ac28d2a)
